### ISSUE TITLE
Update the BlueCompute DevOps pipeline with latest OpenShift or Tekton

### ISSUE THREAD

**Issue by gangchen03** [ibm-cloud-architecture/refarch-cloudnative-kubernetes#222]
The BlueCompute existing DevOps pipeline is built on community Jenkins, we need to consider:

- Using OpenShift Jenkins
- Using Tekton.

Need to work with Hollis.

**Comment by yeekangc** [comment]
+1 for Tekton/OpenShift Pipelines.

FYI, @dewan-ahmed.

**Comment by dewan-ahmed** [comment]
+1 for Tekton/OpenShift Pipelines as well.

@gangchen03 where can we get more context on this i.e. what is the original issue and where the effort sits currently?

**Comment by yeekangc** [comment]
Dewan, FYI that refarch-cloudnative-kubernetes is our reference architecture/implementation for microservices on Kubernetes.  You can find out more at the top-level repo or from https://www.ibm.com/cloud/architecture/architectures/microservices.

I will defer to Gang to chime in and sounds like Gang and team are looking at potential updates to cover the latest technologies.

**Comment by gangchen03** [comment]
yeah, we want to refresh the reference implementation given we have new team members join the team.

**Comment by gangchen03** [comment]
so, the goal is the update the BlueCompute DevOps CI/CD reference implementation (in https://github.com/ibm-cloud-architecture/refarch-cloudnative-devops-kubernetes) to Tektons
